### ISSUE TITLE
Hidden grade items in Single (student) View not shaded lighter

### ISSUE THREAD

**Issue by kimboking** [gjb2048/moodle-theme_essential#487]
In the Categories and Items view of the gradebook, hidden items are customarily shaded lighter which is still the case in Moodle 2.8/Essential 2.8.1.4.  However, in the Single View (for any student), hidden grade items are NOT shaded lighter.  This could potentially cause some confusion or unnecessary grading for the instructor.
![moodle 2 8 grader report hidden items](https://cloud.githubusercontent.com/assets/6333407/7477005/2c769d08-f31f-11e4-989e-3691fe1a3c38.png)
![moodle 2 8 single view hidden items](https://cloud.githubusercontent.com/assets/6333407/7477016/34d49900-f31f-11e4-93db-d55f1e7a8713.png)

**Comment by gjb2048** [comment]
Ok, cannot replicate using the screen on top:

![2015-05-05 17_17_27-grades_ setup](https://cloud.githubusercontent.com/assets/1058419/7477282/b4c106e6-f34a-11e4-9159-364d58623ab2.jpg)

Screen shot 'demonstrating' the issue is not the same page!

**Comment by kimboking** [comment]
The screenshots I provided are 2 separate pages... The Categories and Items page, and the Single View for a student.  On the Categories and Items page, hidden grade items are a lighter shade font.  On the Singe View page for a student, there is no indication that any of the grade items are hidden grade items.  Am I making sense?

**Comment by gjb2048** [comment]
Yes you are, but there is nothing Essential can do about it.  This is a core 'bug' as there is no 'class' being generated where I could determine that the item is hidden, so Essential:

![2015-05-05 17_20_42-grades_ view](https://cloud.githubusercontent.com/assets/1058419/7477431/a542afca-f34b-11e4-8d96-9e07cd48a0b2.jpg)

and the Clean theme:

![2015-05-05 17_21_58-grades_ view](https://cloud.githubusercontent.com/assets/1058419/7477435/afd26976-f34b-11e4-8658-dd7995af1116.jpg)

Therefore, please raise a point on the 'Gradebook' forum: https://moodle.org/mod/forum/view.php?id=2122 or search for / raise a core Moodle tracker issue: https://tracker.moodle.org/secure/Dashboard.jspa - as core code needs to change before I can do anything.

**Comment by kimboking** [comment]
Got it. Will do.  Thanks for taking time to check into this!

**Comment by gjb2048** [comment]
What is the status of this?
